site/public/css/authentication.css

```diff
@@ -50,3 +50,45 @@
 #saml-login {
     color: var(--btn-text-white);
 }
+
+.hidden-helper {
+    display: none;
+    padding: 10px;
+    font-size: small;
+}
+
+.display-box input {
+    display: block;
+    width: 100%;
+}
+
+.requirements-button {
+    display: inline-block;
+    font-weight: 400;
+    color: #212529;
+    text-align: center;
+    vertical-align: middle;
+    background-color: transparent;
+    border: 1px solid;
+    padding: 0.3rem;
+    font-size: x-small;
+    line-height: 1;
+    border-radius: 0.25rem;
+    transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+}
+  
+
+@media (prefers-color-scheme: dark) {
```

Comment:
Why is there more styling in dark mode beyond just basic color switching?

Reply:
Most of the CSS was here before I took over the PR, and didn't look into it much. I will look over it this weekend.